tools/fastp/fastp.xml

```diff
@@ -0,0 +1,332 @@
+<tool id="fastp" name="fastp" version="0.12.3">
```

Comment:
The tool wrapper could be "0.12.3.1"  so that updates to the wrapper would not have to coincide with updates to the conda package.

This could be placed into a [token](http://planemo.readthedocs.io/en/latest/writing_advanced.html#macro-tokens) to simplify things.  (see lines 1-12 [here](https://github.com/galaxyproject/tools-iuc/pull/1597/files#diff-2293f1039f66cfd118915415edaf6538R1))